src/styles/tools/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$transition-duration: 0.3s;
+$transition-property: color;
```

Comment:
It's okay just to pass property every time, no need to save one particular prop in variable, but okay

I think better solution will be mixin that will take property as parameter, or placeholder. Or at least change name of this variable 🙂 It's definitely not clear what is property stored here